core/src/main/java/org/vertexium/ElementBase.java

```diff
@@ -525,4 +525,12 @@ protected boolean isIncluded(HistoricalPropertyValue pv) {
     public EnumSet<FetchHint> getFetchHints() {
         return fetchHints;
     }
+
+    @Override
+    public int compareTo(Object o) {
```

Comment:
Since you've provided a default implementation in the interface, is this needed?